trace_decoder/src/compact/compact_prestate_processing.rs

```diff
@@ -697,6 +742,7 @@ impl<C: CompactCursor> WitnessBytes<C> {
         (instr_and_bytes_buf.into(), res)
     }
 
+    #[allow(dead_code)]
     fn process_into_instructions_and_keep_bytes_parsed_to_instruction_and_bail_on_first_failure_intern(
```

Comment:
Do we plan on using it? Otherwise I'd just go for removing it

Reply:
Yeah same thoughts as above.